recipe/meta.yaml

```diff
@@ -1,20 +1,20 @@
 {% set name = "Pint" %}
-{% set version = "0.11" %}
-{% set sha256 = "308f1070500e102f83b6adfca6db53debfce2ffc5d3cbe3f6c367da359b5cf4d" %}
+{% set version = "0.13" %}
+{% set sha256 = "13fa35ffa3ddf53161b4304917085687950e0de461bcdd4d65cdee11a43d7fb2" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  fn: {{name}}-{{version}}.tar.gz
-  url: https://pypi.io/packages/source/{{name[0]}}/{{name}}/{{name}}-{{version}}.tar.gz
-  sha256: {{sha256}}
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
 
 build:
-    noarch: python
-    script: SETUPTOOLS_SCM_PRETEND_VERSION={{ version }} {{ PYTHON }} -m pip install . -v
-    number: 1
+  noarch: python
+  script: SETUPTOOLS_SCM_PRETEND_VERSION={{ version }} {{ PYTHON }} -m pip install . -v
+  number: 0
 
 requirements:
```

Comment:
@dopplershift: `packaging` (and `importlib-metadata` for `python < 3.8`) recently became a runtime dependency of `pint`, so adding that to `requirements.run` should make the CI pass.